src/routes/NavBar.svelte

```diff
@@ -10,62 +10,125 @@
     function closeMenu() {
         mobileMenu = false;
     }
+
```

Comment:
```suggestion

```

I know this is a nitpick, but see prior comment. Formatter would normally flag this.